mm2src/lp_ordermatch.rs

```diff
@@ -561,13 +580,21 @@ struct GetOrderbookRes {
     pubkey_orders: HashMap<String, GetOrderbookPubkeyItem>,
     #[serde(default)]
     protocol_infos: HashMap<Uuid, BaseRelProtocolInfo>,
+    #[serde(default)]
+    conf_infos: HashMap<Uuid, OrderConfirmationsSettings>,
 }
 
+#[allow(clippy::type_complexity)]
```

Comment:
Similar to https://github.com/KomodoPlatform/atomicDEX-API/pull/1190#discussion_r805882490 - please simplify the return type (I guess a struct is needed) instead of adding `allow` directive.